### PR TITLE
Feature/update script documentation

### PR DESCRIPTION
**Pull request by rhodesn**
Documentation for #157.

### CODE REVIEW

**Comment by rhodesn** [comment]
Trimmed output:
```
DIRECTORIES
       Default  Directory locations are listed are listed below; custom paths can be specified in the config
       file. The use of additional, none default, config files is required when running more than one  auter
       profile.

   Pre/Post Hooks
       If any scripts in the pre/post hook directories are not executable, they will be skipped by auter and
       a warning containing the filename will be logged. If any scripts in  the  pre/post  hook  directories
       exit  with  a  non-zero  exit code an error will be logged with the filename and exit code, and auter
       will abort.

       /etc/auter/pre-prep.d
              Directory containing scripts to be executed before downloading updates

       /etc/auter/post-prep.d
              Directory containing scripts to be executed after downloading updates

       /etc/auter/pre-apply.d
              Directory containing scripts to be executed before applying updates

       /etc/auter/post-apply.d
              Directory containing scripts to be executed after applying updates

       /etc/auter/pre-reboot.d
              Directory containing scripts to be executed before rebooting

       /etc/auter/post-reboot.d
              Directory containing scripts to be executed after rebooting

              There is a hard coded delay of 5 minutes post-reboot to allow the system to become fully ready
              before the post-reboot scripts are executed.

   Hook execution order
       Files in the pre/post script directories are parsed in sorted lexical order. That is, /etc/auter/pre-
       apply.d/01_first will be parsed before /etc/auter/pre-apply.d/10_second. Be aware  that  because  the
       sorting  is  lexical,  not  numeric,  /etc/auter/pre-apply.d/1_whoops would be loaded after /etc/pre-
       apply.d/10_second. Using a consistent number of leading zeroes in the file names can be used to avoid
       such problems. For example the following execution order would occur for pre-apply scripts:

              pre-apply.d/00-first.sh
              pre-apply.d/99-second.sh
              pre-apply.d/AA-fourth.sh
              pre-apply.d/aa-third.sh
              pre-apply.d/zz-fifth.sh
              pre-apply.d/ZZ-sixth.sh

FILES
       /etc/auter/auter.conf
              Default  config  file  location. Use the --config option to pass an alternative config file to
              use within /etc/auter.

       /etc/cron.d/auter
              The default cron file with examples

       /usr/bin/auter
              The main auter script which contains linux generic code

       /usr/lib/auter/auter.module
              The auter module for the relevant package manager

       /usr/share/doc/auter-0.11/LICENSE
              The Apache License

       /usr/share/doc/auter-0.11/MAINTAINERS.md
              A list of maintainers and mailing list

       /usr/share/doc/auter-0.11/NEWS
              The NEWS file and changelog for auter

```

**Comment by rippiedoos** [comment]
Nice documentation. I would suggest updating the documentation in the contrib-folder (auter/contrib/README.md) as well to reflect these changes.

File: auter.help2man-sections [hunk 1]
```diff
@@ -1,64 +1,82 @@
-[directories]
-Default Directory locations are listed, but also customizable in the config file. The customization will be required for the use of multiple auter profiles
+[Directories]
 
-If any scripts in the pre/post hook directories are not executable a warning will be logged with  the  filename  but auter  will continue.
+Default Directory locations are listed are listed below; custom paths can be specified in the config file. The use of additional, none default, config files is required when running more than one auter profile.
```
Comment (reaperzn) [inline auter.help2man-sections]: ```locations are listed are listed below```
copy/paste error?

```use of additional, none default, config files```
Should this be ```non-default```?

**Review by reaperzn — CHANGES_REQUESTED** [review]
Just some minor issues highlighted. I also agree that we should add a note to the contrib README to include a line about script names.

**Review by reaperzn — APPROVED** [review]

**Comment by reaperzn** [comment]
Looks good. Thanks @nrhodes91

**Pull request merged**